docs-source/spring/content/infrastructure/networking/_index.md

```diff
@@ -0,0 +1,129 @@
+
+> For **custom installations**, including On-Premises, it is the responsibility of the customer to ensure network access controls to provide both operational access and security.  The Oracle Cloud Infrastructure (OCI) **standard installation** setup can be used as a general template.
+
+# Network Infrastructure Overview
+
+![OCI Network Infrastructure](images/network_infra.png "OCI Network Infrastructure")
+
+## OCI Network Infrastructure
+
+All infrastructure resources are split between two subnets: one public and one private subnet with access restricted by Network Security Groups (Firewalls).  All access is blocked by default with specific IP and Port opened, as documented below, for proper operation.
+
+> Note minimum CIDRs are based on an Oracle Cloud Infrastructure (OCI) deployments.  Other cloud vendors reserve usable IPs within a CIDR reducing the usable IPs that can be used by the infrastructure.
+
+### Public Subnet
+
+The public subnet allows external access to the Oracle Backend for Spring Boot and Microservices resources.  Ingress/Egress to the Public Subnet and from the Internet is provided by an Internet Gateway.
+
+At a minimum, the public subnet requires two usable IP Addresses (`/30`).
+
+ICMP traffic is allowed between the Public and Private Subnets for resource discovery. 
+
+#### Load Balancer
+
+An external Load Balancer is used in conjunction with Ingress resources to expose Services to the internet.  Ingress to the Load Balancer should be restricted by CIDR and Ports (default: `0.0.0.0/0`; `80`,`443`).  All TLS communications terminate at the LoadBalancer.  For more information on the certificate for TLS, please follow [OCI Load Balancer TLS Certificate Setup](#oci-loadbalancer-tls-certificate-setup).
+
+#### Kubernetes API Endpoint
+
+The Kubernetes API Endpoint, within the Control Plane, is used to manage the Kubernetes Cluster as well as providing access to Kubernetes services that are not exposed to the Load Balancer via Port Forwarding.  Ingress to the Endpoint is restricted to port `6443` and should be further restricted by CIDR (default: `0.0.0.0/0`).
+
+In Oracle Cloud Infrastructure (OCI), the Oracle Kubernetes Engine (OKE) Control Plane is allowed egress to OCI services via port `443`.  Access to these services are required for the OKE Managed Control Plane.
+
+### Private Subnet
+
+The private subnet isolates its resources from direct external access.  Ingress to resources in the private subnet is restricted to the Load Balancer and Kubernetes API Endpoint via specific Network Security Group (Firewall) rules.  Optionally, a Bastion Service, within the Private subnet, can provide additional non-restricted access to the resources in the Private Network.
+
+Egress to the Internet is provided by a NAT Gateway, while Egress to other Cloud Services (such as a Container Repository) is provided by a Service Gateway.
+
```

Comment:
right about now i started wishing for a diagram of this :)

Reply:
more diagram apart from the one at the top?